doc/user/content/console/data.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Data (Object Explorer)"
```

Comment:
In our changelog entry, I believe we called this feature "Database Object Explorer", probably good to be consistent.

Reply:
👍 Can update to Database Object Explorer -- I had gone with "Data (...)" since the menu item in the console just stated Data (but not too much of a leap on users to see Database Object Explorer).